### DISCUSSION TITLE
Any plans to release Textarea control

### DISCUSSION THREAD

**Discussion by manupatel007** [flet-dev/flet#1981] [General]
Trying to create a text editor in windows with special capabilities. Searched for textarea component, seems like it is missing. Tried to stretch textfield vertically and horizontally, but unable to stretch it vertically. Also, not possible to go to new line in textfield. Any idea on how to achieve this or introducing it in some upcoming release.

**Comment by ndonkoHenri** [comment]
> Also, not possible to go to new line in textfield. 

You can use the [`multiline`](https://flet.dev/docs/controls/textfield/#multiline) prop for that.
Can that be of help?

**Reply by manupatel007** [reply]
I see, thanks a lot for helping out. Tried out and it works smoothly as per my requirement.
